tpchgen-arrow/tests/reparse.rs

```diff
@@ -0,0 +1,100 @@
+//! Verifies the correctness of the Arrow TPCH generator by parsing the canonical TBL format
+//! and comparing with the generated Arrow RecordBatches
+
+use arrow::array::RecordBatch;
+use arrow::datatypes::SchemaRef;
+use std::io::Write;
+use std::sync::Arc;
+use tpchgen::csv::LineItemCsv;
+use tpchgen::generators::{LineItem, LineItemGenerator};
+use tpchgen_arrow::{LineItemArrow, RecordBatchIterator};
+
+#[test]
+fn test_tpchgen_lineitem_tbl() {
```

Comment:
I am quite pleased with these tests to check correctness by checking consistency with the tbl output format. I think they would also catch the issues @niebayes has found in https://github.com/clflushopt/tpchgen-rs/issues/65 when I implement them for the other tables